src/main/java/mate/academy/Event.java

```diff
@@ -1,4 +1,7 @@
 package mate.academy;
 
-public record Event(String type, Object source) {
+public record Event(
```

Comment:
Consider providing a more detailed documentation comment for the `Event` record. This would help other developers understand the purpose of this record and when it should be used.

Reply:
Thank you for the amazing advice!